fhir-tools/src/main/java/com/ibm/fhir/tools/CodeGenerator.java

```diff
@@ -3285,6 +3381,14 @@ private boolean isBackboneElement(JsonObject elementDefinition) {
         List<JsonObject> types = getTypes(elementDefinition);
         return !types.isEmpty() && "BackboneElement".equals(types.get(0).getString("code", null));
     }
+    
+    private boolean isBackboneSubtype(String className) {
```

Comment:
If I follow this correctly, it'll tell you if it's 

com.ibm.fhir.model.type.test.test
vs
com.ibm.fhir.model.type.test2

com.ibm.fhir.model.other.test3

test2 would return false? 
test1 would return true
test 3 would return true?

Reply:
specifically what we're trying to avoid here is having visit methods for every single *nested* BackboneElement subtype in the model.

the only reason for stripping the `com.ibm.fhir.model.type` is because we want to avoid fully qualified types like `com.ibm.fhir.model.type.String` from coming back true.

com.ibm.fhir.model.type.Test.Test1 would return true (and thus get skipped)
com.ibm.fhir.model.type.Test2 would return false
com.ibm.fhir.model.resource.Test3 would return true, but I'm not sure it matters much since this is a private method thats only called with fullyqualified names in the com.ibm.fhir.model.type package

Reply:
i'll just add some javadoc to explain to expected usage

Reply:
Cool.  I was attempting to follow :)

Reply:
updated